uberfire-nio2-backport/uberfire-nio2-impls/uberfire-nio2-jgit/src/main/java/org/uberfire/java/nio/fs/jgit/util/commands/CreateRepository.java

```diff
@@ -77,6 +78,17 @@ public CreateRepository(final File repoDir,
                     .build();
 
             final org.eclipse.jgit.api.Git git = new org.eclipse.jgit.api.Git(repo);
+            
+            String ssl =  System.getProperty("org.uberfire.nio.git.http.sslVerify"); 
```

Comment:
This property should be placed here: https://github.com/kiegroup/appformer/blob/dd03638ed7aba6146e042b7a7062d9ad4630bd3f/uberfire-nio2-backport/uberfire-nio2-impls/uberfire-nio2-jgit/src/main/java/org/uberfire/java/nio/fs/jgit/JGitFileSystemProviderConfiguration.java#L29